src/web/browser/sentryLoader/sentry.ts

```diff
@@ -0,0 +1,62 @@
+import * as Sentry from '@sentry/browser';
```

Comment:
This file was broadly unchanged. I removed the `initialiseSentry` function that the code was wrapped in and we no longer do anything with samplig or ad block detection here

Reply:
It seems git can find it easier to identify a file rename if you separate the change and move into distinct commits.

https://stackoverflow.com/questions/433111/how-to-make-git-mark-a-deleted-and-a-new-file-as-a-file-move